### PR TITLE
added a section to Readme to show how to check account balances

### PR DESCRIPTION
**Pull request by ryancwalsh**
See https://github.com/near/workspaces-rs/issues/172

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,6 +123,62 @@ Then later on, we can view our minted NFT's metadata via our `view` call into `n
...
+        )
+        .args_json(json!({"recipient": &recipient.id()}))?
+        .gas(GAS_FOR_ACCOUNT_CALLBACK.0)
+        .deposit(transfer_amount)
+        .transact()
+        .await?;
+    assert_eq!(
+        recipient.view_account(&worker).await?.balance,
+        initial_balance + transfer_amount
+    );
```
[the remark this answers is not in the document]
Comment (ryancwalsh) [reply README.md:177]: Totally agree. Great point.

File: README.md [hunk 2]
```diff
@@ -123,6 +123,62 @@ Then later on, we can view our minted NFT's metadata via our `view` call into `n
...
+async fn test_some_function_that_involves_a_transfer() -> anyhow::Result<()> {
+    let initial_balance: Balance = near_units::near::parse("1").unwrap();
+    let transfer_amount: Balance = near_units::near::parse("0.1").unwrap();
+    let worker = workspaces::sandbox().await?;
+    let contract = worker
+        .dev_deploy(&include_bytes!("../target/res/your_project_name.wasm").to_vec())
+        .await?;
+    contract
+        .call(&worker, "new")
+        .gas(GAS_FOR_ACCOUNT_CALLBACK.0)
```
[the remark this answers is not in the document]
Comment (ryancwalsh) [reply README.md:142]: Cool. I didn't know about this but will add this now. Thanks!

**Comment by ryancwalsh** [comment]
@ChaoticTempest Ok, what is shown in the Readme now works for me locally when I run it against this function:

```rs
pub fn some_function_that_involves_a_transfer(
        &self,
        destination_account: &AccountId,
    ) -> Promise {
        let amount: Amount = env::attached_deposit();
        Promise::new(destination_account.clone()).transfer(amount) 
    }
```

**Comment by ryancwalsh** [comment]
@ChaoticTempest Are any other changes needed before accepting and merging this PR? Thanks.

**Review by ChaoticTempest — APPROVED** [review]

**Pull request merged**